authors/settings.py

```diff
@@ -179,3 +179,11 @@
 #Update Database Configuration
 prod_db  =  dj_database_url.config(conn_max_age=500)
 DATABASES['default'].update(prod_db)
+
+#EMAIL CONFIGURATIONS
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_USE_TLS = True
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'thy.realers@gmail.com'
```

Comment:
I suggest you make the `EMAIL_HOST_USER` be picked from the environment also. For easy switching when the need arises.

Reply:
@HabibSentongo this is a nice suggestion. Let me implement it.

Reply:
@HabibSentongo  I have worked on this.